pyproject.toml

```diff
@@ -14,59 +13,56 @@ dependencies = [
     "click<9,>=8",
     "curies",
     "jinja2<4,>=3",
-    "linkml-runtime>=1.7.2",
-    "pydantic>=2.0.0",
+    "lark<2,>=1",
+    "linkml-runtime>=1.9.1",
```

Comment:
Deps are updated to the versions installed by running `uv sync` on the existing pyproject.toml file.